Source/Core/Core/Slippi/SlippiUser.cpp

```diff
@@ -276,7 +276,7 @@ void SlippiUser::FileListenThread()
 		if (AttemptLogin())
 		{
 			runThread = false;
-			main_frame->RaiseRenderWindow();
+			// main_frame->RaiseRenderWindow();
```

Comment:
macOS still uses this, just fyi - so maybe instead of commenting it out just block it off with an `#ifdef __APPLE_`.

Reply:
Issue is that the nogui build will still fail since we don't compile DolphinWX at all in nogui.

Reply:
I think I have an idea for a fix...

Reply:
Well, sure - `#ifndef NOGUI` or whatever on the imports and raise/lower calls would suffice, no?